pkg/scanners/terraform/executor/statistics.go

```diff
@@ -12,27 +14,54 @@ import (
 )
 
 type StatisticsItem struct {
-	RuleID          string
-	RuleDescription string
-	Links           []string
-	Count           int
+	RuleID          string   `json:"rule_id"`
+	RuleDescription string   `json:"rule_description"`
+	Links           []string `json:"links"`
+	Count           int      `json:"count"`
 }
 
 type Statistics []StatisticsItem
 
+type StatisticsResult struct {
+	Result Statistics `json:"results"`
+}
+
 func SortStatistics(statistics Statistics) Statistics {
 	sort.Slice(statistics, func(i, j int) bool {
 		return statistics[i].Count > statistics[j].Count
 	})
 	return statistics
 }
 
-func (statistics Statistics) PrintStatisticsTable(w io.Writer) {
-	table := tablewriter.NewWriter(w)
+func (statistics Statistics) PrintStatisticsTable(format string, w io.Writer) error {
+	// lovely is the default so we keep it like that
+	if format != "lovely" && format != "markdown" && format != "json" {
+		return fmt.Errorf("you must specify only lovely, markdown or json format with --run-statistics")
+	}
+
 	sorted := SortStatistics(statistics)
+
+	if format == "json" {
+		result := StatisticsResult{Result: sorted}
+		val, err := json.MarshalIndent(result, "", "    ")
+		if err != nil {
+			return err
+		}
+
+		fmt.Println(string(val))
```

Comment:
We have a writer to send the output to here, i.e.
```suggestion
		_, _ = fmt.Fprintln(w, string(val))
```